interpreter/cling/lib/Interpreter/BackendPasses.cpp

```diff
@@ -16,9 +16,10 @@
 #include "llvm/Analysis/InlineCost.h"
 #include "llvm/Analysis/TargetLibraryInfo.h"
 #include "llvm/Analysis/TargetTransformInfo.h"
-#include "llvm/IR/LegacyPassManager.h"
 #include "llvm/IR/Module.h"
 #include "llvm/IR/Verifier.h"
+#include "llvm/Passes/PassBuilder.h"
+#include "llvm/Passes/PassPlugin.h"
```

Comment:
Do we need `llvm/Passes/PassPlugin.h`? Instead, I think we should definitely include `llvm/IR/PassManager.h` because it defines things such as `PassInfoMixin`.

Reply:
Thanks for the review. 
I added that to try loading the SYCL passes in cling. Removed!